js/main.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-undef */
+
+// player factory for create player objects
+const playerFactory = (id, name, token) => ({
+  id,
+  name,
+  token,
+});
+
+let counter = 0;
```

Comment:
we shouldn't use global variable right 😉 
It doesn't use `global` code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))

Reply:
done